src/v2/Components/ArtworkFilter/ArtworkFilters/PriceRangeFilterNew.tsx

```diff
@@ -128,39 +128,35 @@ export const PriceRangeFilterNew: FC<PriceRangeFilterNewProps> = ({
         </Box>
       </Flex>
 
-      <Box mt={2} mx={`${RANGE_DOT_SIZE / 2}px`}>
-        {bars.length > 0 && !isAllBarsEmpty ? (
-          <Histogram
-            bars={bars}
-            selectedRange={[sliderRange[0], sliderRange[1]]}
-            data-testid="PriceFilterHistogram"
-          />
-        ) : null}
-
-        <Spacer pb={2} />
```

Comment:
Never use padding on `Spacer` components since they wouldn't collapse. (Making a less error prone `Spacer` is on my TODOs this week).